lib/queries/add.js

```diff
@@ -13,7 +13,11 @@ module.exports = function (objectToAdd) {
     return objectToAdd[field];
   });
 
-  return 'INSERT INTO challenges ('
-    + escapeFields.concat(unescapeFields).join(', ') + ')'
-    + ' VALUES (' + escapedValues.concat(unescapedValues).join(', ') + ');';
+  return [
+    'INSERT INTO challenges (',
+    escapeFields.concat(unescapeFields).join(', '),
+    ') VALUES (',
+    escapedValues.concat(unescapedValues).join(', '),
+    ') RETURNING id;'
```

Comment:
I think id for now is fine - we just need it to go from /challenges/add to /challenges/{id}/tags (which is the flow for adding a new challenge)